### DISCUSSION TITLE
question about eslint Eclipse copyright header comment

### DISCUSSION THREAD

**Discussion by rileydanejohnston** [eclipse-glsp/glsp#881] [General]
Hello,
My company is using GLSP in a product it's releasing soon. I'm trying to figure out if there are legal ramifications to overriding the eslint header (basically the EclipseSource copyright comment). Is this something I can remove and replace with my own header? Thanks!

**Comment by planger** [comment]
Hi,

sure, you own the copyright of the files you wrote and thus can put your copyright header into it. So it only makes sense to change the eslint config in your project to avoid eslint to complain for your files that have your copyright header.

For files that you took from GLSP as is or that you just modified (derivative work), you must keep the original copyright header.
Disclaimer: I'm not a lawyer though ;-)

As you are about to release a product based on GLSP, we'd be thrilled if your company would be listed on the [ECD Tools website adopter list](https://ecdtools.eclipse.org/adopters/). It is a very easy process to get listed there. Also, we'd be happy to show a screenshot of your tool in the [GLSP gallery](https://www.eclipse.org/glsp/gallery/), if possible?

Thanks!
